inc/shortcodes/glossary/class-glossary.php

```diff
@@ -177,12 +189,29 @@ function glossaryTerms() {
 		return $output;
 	}
 
+	/**
+	 * @param string $needle
+	 * @param string $haystack
+	 *
+	 * @return bool
+	 */
+	public function commaDelimitedStringSearch( $needle, $haystack ) {
```

Comment:
Should this be in `\Pressbooks\Utility`? Seems like it might be useful elsewhere in the future.